### PR TITLE
add Platform-Support.adoc and polish other doco

### PR DESCRIPTION
**Pull request by JerryShea**

### CODE REVIEW

File: docs/Platform-Support.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+* Linux on Intel for production
+* Windows & MacOS Intel and M1 for development - not all functionality fully supported
+* 32/64-bit ARM - not all functionality fully supported
+
+== C++ & Python
+
+Chronicle C++ and Python libraries support
+
+* Linux on Intel for production
+* MacOS on Intel and M1 for production
```
Comment (rogersimmons) [inline docs/Platform-Support.adoc]: Linux on x86_64 for production
macOS on x86_64 and arm64 for production

File: docs/Java-Version-Support.adoc [hunk 2]
```diff
@@ -1,5 +1,5 @@
 = Java Version Support
-Peter Lawrey
+Peter Lawrey, Jerry Shea
```
Comment (rogersimmons) [inline docs/Java-Version-Support.adoc]: suggest we remove authorship hooks altogether in general (as above for Anatomy.adoc)

File: docs/Platform-Support.adoc [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+== Rust
+
+Chronicle Rust libraries
+
+* Linux on Intel for production
```
Comment (rogersimmons) [inline docs/Platform-Support.adoc]: Linux on x86_64 for production

**Review by rogersimmons — CHANGES_REQUESTED** [review]

File: docs/Java-Version-Support.adoc (hunk 2 above)
Comment (JerryShea) [reply docs/Java-Version-Support.adoc]: Yep - fair enough

File: docs/Platform-Support.adoc [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+Chronicle C++ and Python libraries support
+
+* Linux on Intel x86_64 for production
+* MacOS on Intel x86_64 and arm64 (M1) for production
+
+== Rust
+
+Chronicle Rust libraries
+
+* Linux on Intel for production
```
Comment (rogersimmons) [inline docs/Platform-Support.adoc]: Intel x86_64
(Need the x86_64 piece in there, as no 32-bit support)

File: docs/Platform-Support.adoc [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+* Windows & MacOS Intel and M1 for development - not all functionality fully supported
+* 32/64-bit ARM - not all functionality fully supported
+
+== C++ & Python
+
+Chronicle C++ and Python libraries support
+
+* Linux on Intel x86_64 for production
+* MacOS on Intel x86_64 and arm64 (M1) for production
```
Comment (rogersimmons) [inline docs/Platform-Support.adoc]: more than just the M1. Maybe generically refer to Apple Silicon?

**Review by rogersimmons — CHANGES_REQUESTED** [review]

**Review by rogersimmons — APPROVED** [review]

**Pull request merged**